.golangci.yml

```diff
@@ -12,25 +12,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+version: "2"
+run:
+  issues-exit-code: 1
 linters:
   enable:
-  - errcheck
-  - gofmt
-  - goimports
-  - gosec
-  - gocritic
-  - misspell
-  - revive
-  - unused
-issues:
-  exclude-rules:
-  - path: _test\.go
-    linters:
     - errcheck
+    - gocritic
     - gosec
+    - misspell
+    - revive
+    - unused
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+    rules:
+      - linters:
+          - errcheck
+          - gosec
+        path: _test\.go
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
```

Comment:
I can remove these as well -- not sure why the migrate command adds these: maybe they are typical for golang projects?